<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-02-05T20:24:11Z" name="ExceptionTestSuite" finished-at="2019-02-05T20:24:34Z" duration-ms="23586">
    <groups>
    </groups>
    <test started-at="2019-02-05T20:24:11Z" name="ExceptionTest" finished-at="2019-02-05T20:24:34Z" duration-ms="23586">
      <class name="com.herokuapp.theinternet.ExceptionTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.herokuapp.theinternet.ExceptionTest@49e5f737]" started-at="2019-02-05T20:24:11Z" name="setUp" finished-at="2019-02-05T20:24:21Z" duration-ms="10278" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="notVisibleTest()[pri:0, instance:com.herokuapp.theinternet.ExceptionTest@49e5f737]" started-at="2019-02-05T20:24:21Z" name="notVisibleTest" finished-at="2019-02-05T20:24:30Z" duration-ms="9498" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- notVisibleTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.herokuapp.theinternet.ExceptionTest@49e5f737]" started-at="2019-02-05T20:24:30Z" name="tearDown" finished-at="2019-02-05T20:24:34Z" duration-ms="3748" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.herokuapp.theinternet.ExceptionTest -->
    </test> <!-- ExceptionTest -->
  </suite> <!-- ExceptionTestSuite -->
</testng-results>
